.github/workflows/test.yaml

```diff
@@ -14,14 +14,31 @@ jobs:
       # version of go.
       PATH: /snap/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games
       GOROOT: ""
-
+      # XXX: only skip format checks on 1.13+
+      SKIP_GOFMT: 1
+    strategy:
+      # we cache successful runs so it's fine to keep going
+      fail-fast: false      
```

Comment:
This was meant to prevent a bug in spread where spread refuses to stop and we waste money. Please revert this and keep the original comment.